crowbar_framework/app/models/node_object.rb

```diff
@@ -1138,6 +1138,22 @@ def ssh_cmd(cmd)
     end
   end
 
+  def net_rpc_cmd(cmd)
+    case cmd
+    when :power_cycle
+      unless system("net", "rpc", "shutdown", "-f", "-r", "-I", @node.name ,"-U", "Administrator%#{@node[:provisioner][:windows][:admin_password]}")
```

Comment:
Something like `search(:node, "platform:windows")`. You can find plenty of search examples in the cookbooks we already have...